client/state/posts/utils.js

```diff
@@ -31,7 +31,9 @@ import stripHtml from 'lib/post-normalizer/rule-strip-html';
  * Constants
  */
 
-const REGEXP_SERIALIZED_QUERY = /^((\d+):)?(.*)$/;
+const REGEXP_SERIALIZED_QUERY = /^((\d+):)?(.*)$/,
+	REGEXP_EMPTY_CONTENT = /^<p>(<br[^>]*>|&nbsp;|\s)*<\/p>$/,
```

Comment:
Thanks @youknowriad and @aduth for the clarification. I always like to ask when we have to make assumptions to understand the code. Maybe it would be helpful to clarify this in a comment and additionally throw in that snippet from @aduth showing what we might expect to be empty.